### ISSUE TITLE
Webform description text not adapting to section background

### ISSUE THREAD

**Issue by cory-skeers** [uiowa/uiowa#4385]
# Problem
Field descriptions in webforms are not adapting to section backgrounds, leading to possible accessibility issues (eg black description text on a black background.

![image](https://user-images.githubusercontent.com/49074566/139731158-8120a23e-53c8-4732-b6ae-86390c379ce9.png)

# Steps to reproduce

Create a new webform, or edit an existing webform to add a description.
Add the webform to sections with varying background selections.

# Proposed solution

Take a look at how field label, button, etc colors are being updated in `background.scss`, `forms.scss`, or elsewhere, and apply to descriptions as well.
